resources/js/components/BarChart.vue

```diff
@@ -110,6 +111,7 @@
           scales: {
             yAxes: [{
               ticks: {
+                min: yAxesMin,
```

Comment:
it throw an error `yAxesMin is not defined`
Can you help to change it with `min: this.card.options.yAxesMin != undefined ? this.card.options.yAxesMin : undefined,` ?

Thank you